src/index.js

```diff
@@ -375,7 +375,7 @@ export default class Bili extends EventEmitter {
             // Check if the module exists
             resolveFrom.silent(process.cwd(), source) === null
           ) {
-            logger.warn(`Module "${source}" was not installed, you may run "${chalk.cyan(`${getPackageManager()} add ${source}`)}" to install it!`)
+            logger.warn(`Module "${source}" was not installed, you may run "${chalk.cyan(`${getPackageManager() === 'yarn' ? 'yarn add' : 'npm install'} ${source}`)}" to install it!`)
```

Comment:
well `npm add` also works 😂

Reply:
oh seems it's working for real :laughing: , why really this even works !!??,
There is no alias for that in [docs](https://docs.npmjs.com/cli/install)
Even if it works it's not official

Reply:
![screenshot_20181116-192817__01](https://user-images.githubusercontent.com/8784712/48618978-ee4b0580-e9d5-11e8-9b2f-3955df43c55e.jpg)

does this count 🤔